### PR TITLE
[Magiclysm] Adds new zombie animals 

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Mods "[Magiclysm] Adds new zombie animals"

#### Purpose of change
Brings forward the closed PR that was pretty much complete by Saicchi.  

#### Describe the solution
Adds new zombies, zombify into, and the mutant tainted meat is added to vanilla so it can be used in vanilla places where appropriate.  Pulls everything but zombie goblins from #48755.  Fixes winter wolves not spawning and zombie werewolves not spawning.

#### Describe alternatives you've considered
None

### CODE REVIEW

**Comment by Theawesomeboophis** [comment]
Nice, it's great to see this idea revived, more content is always nice.

File: data/mods/Magiclysm/monsters/zombified_monsters.json [hunk 1]
```diff
@@ -61,5 +61,163 @@
...
+    "melee_dice_sides": 6,
+    "melee_cut": 6,
+    "dodge": 1,
+    "armor_bash": 5,
+    "armor_cut": 2,
+    "armor_bullet": 2,
+    "vision_night": 13,
+    "path_settings": { "max_dist": 12 },
+    "harvest": "zombie_owlbear",
+    "death_function": [ "NORMAL" ],
```
Comment (Maleclypse) [inline data/mods/Magiclysm/monsters/zombified_monsters.json:94]:
```suggestion
```

File: data/mods/Magiclysm/monsters/zombified_monsters.json [hunk 2]
```diff
@@ -61,5 +61,163 @@
...
+    "melee_cut": 2,
+    "dodge": 3,
+    "armor_bash": 4,
+    "armor_cut": 1,
+    "armor_bullet": 4,
+    "vision_day": 30,
+    "vision_night": 10,
+    "path_settings": { "max_dist": 3, "avoid_traps": false, "avoid_sharp": false },
+    "harvest": "zombie_fur",
+    "death_function": [ "NORMAL" ],
```
Comment (Maleclypse) [inline data/mods/Magiclysm/monsters/zombified_monsters.json:194]:
```suggestion
```

File: data/json/items/comestibles/carnivore.json [hunk 3]
```diff
@@ -307,6 +307,46 @@
...
+    "price_postapoc": 5,
+    "fun": -15,
+    "flags": [ "TRADER_AVOID", "SMOKABLE", "BAD_TASTE", "RAW" ],
+    "smoking_result": "meat_mutant_tainted_smoked"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "meat_mutant_tainted_cooked",
+    "copy-from": "meat_mutant_tainted",
+    "name": { "str": "cooked tainted mutant meat" },
```
Comment (CountAlex) [inline data/json/items/comestibles/carnivore.json:331]: Probably should use "str_sp".

File: data/json/items/comestibles/carnivore.json [hunk 4]
```diff
@@ -307,6 +307,46 @@
...
+    "parasites": 0,
+    "proportional": { "price_postapoc": 1.5, "fun": 0.7 },
+    "extend": { "flags": [ "EATEN_HOT" ] },
+    "delete": { "flags": [ "RAW" ] }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "meat_mutant_tainted_smoked",
+    "copy-from": "meat_mutant_tainted_cooked",
+    "name": { "str": "smoked tainted mutant meat" },
```
Comment (CountAlex) [inline data/json/items/comestibles/carnivore.json:343]: Same.

File: data/json/items/comestibles/carnivore.json [hunk 5]
```diff
@@ -307,6 +307,46 @@
...
+    "description": "Rotten mutatant meat that has been heavily smoked for preservation.",
```
Comment (CountAlex) [inline data/json/items/comestibles/carnivore.json:344]: "mutatant" -> "mutant"

**Review by CountAlex — COMMENTED** [review]
Some things to fix in next PRs.

**Pull request merged**